ecl/eclcc/eclcc.cpp

```diff
@@ -2949,8 +2964,9 @@ int EclCC::parseCommandLineOptions(int argc, const char* argv[])
         else if (iter.matchFlag(optSaveQueryArchive, "-qa"))
         {
         }
-        else if (iter.matchOptionText(eclRepoPath, "--repocachepath", false, false))
+        else if (iter.matchOption(tempArg, "--repocachepath"))
```

Comment:
This was invalid function to call - matchOptionText appends the option name as well for some reason.